packages/components/src/components/select-native/select-native.tsx

```diff
@@ -1,47 +1,91 @@
 import classNames from 'classnames';
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { SelectHTMLAttributes } from 'react';
```

Comment:
can we use React. SelectHTMLAttributes? as it is more common approach for our project...

Reply:
ok